modules/javafx.media/src/main/java/com/sun/media/jfxmediaimpl/MediaUtils.java

```diff
@@ -175,17 +184,18 @@ else if ((buf[8] & 0xff) == 0x4D && (buf[9] & 0xff) == 0x50 && (buf[10] & 0xff)
     /**
      * Returns the content type given the file name.
      *
-     * @param filename
+     * @param uri
```

Comment:
should the comment be changed since the argument is uri and not a file name?

Reply:
Fixed.